base/sparse/sparsematrix.jl

```diff
@@ -972,7 +972,7 @@ end # macro
 (.-)(A::Number, B::SparseMatrixCSC) = A .- full(B)
 ( -)(A::Array , B::SparseMatrixCSC) = A  - full(B)
 
-(.*)(A::AbstractArray, B::AbstractArray) = broadcast_zpreserving(*, A, B)
+(.*)(A::AbstractArray, B::AbstractArray) = broadcast_zpreserving(MulFun(), A, B)
```

Comment:
I'm changing this line to `Base.MulFun` in the backport. I'm not sure what's going on with master here but I'm moving on to other backports so not thinking too much about it right this second.